Mixpanel/MixpanelPrivate.h

```diff
@@ -24,6 +20,9 @@
 #import <CoreTelephony/CTCarrier.h>
```

Comment:
why do we need TARGET_OS_IPHONE inside !MIXPANEL_LIMITED_SUPPORT. If we arent in TVOS and Extensions, only Watch and iOS is left, but Watch doesnt get here right?